lib/features/recurring_donations/overview/pages/recurring_donations_overview_page.dart

```diff
@@ -0,0 +1,213 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/core/logging/logging_service.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/recurring_donations/cancel/widgets/cancel_recurring_donation_confirmation_dialog.dart';
+import 'package:givt_app/features/recurring_donations/overview/cubit/recurring_donations_cubit.dart';
+import 'package:givt_app/features/recurring_donations/overview/models/recurring_donation.dart';
+import 'package:givt_app/features/recurring_donations/overview/widgets/recurring_donation_item.dart';
+import 'package:givt_app/utils/app_theme.dart';
+import 'package:go_router/go_router.dart';
+
+class RecurringDonationsOverviewPage extends StatefulWidget {
+  const RecurringDonationsOverviewPage({super.key});
+
+  @override
+  State<RecurringDonationsOverviewPage> createState() =>
+      _RecurringDonationsOverviewPageState();
+}
+
+class _RecurringDonationsOverviewPageState
+    extends State<RecurringDonationsOverviewPage> {
+  RecurringDonation? selectedRecurringDonation;
+
+  @override
+  void initState() {
+    super.initState();
+    _fetchRecurringDonations();
+  }
+
+  Future<void> _fetchRecurringDonations() async {
+    await context
+        .read<RecurringDonationsCubit>()
+        .fetchRecurringDonations(context.read<AuthCubit>().state.user.guid);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+
+    return Scaffold(
+      appBar: AppBar(
+        leading: const BackButton(),
+        title: Text(
+          //TODO: POEditor
+          'Recurring donations',
+          style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                fontWeight: FontWeight.bold,
+              ),
+        ),
+      ),
+      body: BlocConsumer<RecurringDonationsCubit, RecurringDonationsState>(
+        listener: (context, state) {
+          LoggingInfo.instance
+              .info('recurring donations cubit state changed on $state');
```

Comment:
Maybe we should think of some standards when it comes to logging, to avoid confusions especially for Tine

Reply:
Agree:) do you have in mind some suggestions?